### ISSUE TITLE
[HOLD] Web - Wrong URL if participant detail page in default room is closed @aneequeahmad

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#10989]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to staging.new.expensify.com and log in with any account
2. Click on any default room (workspace room)
3. Click on header to open details page > settings > go back by clicking back button >
4. Click on members > any member.
5. Click on close button

## Expected Result:
URL should be r/reportId

## Actual Result:
URL is /r/reportId/details

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.2.0.0

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes
 
**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/190176503-5c92f0d5-677f-4ec4-964a-f9713dc6266b.mov

**Expensify/Expensify Issue URL:** 

**Issue reported by:** @aneequeahmad

**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1661463319371319

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by grgia** [comment]
Setting to external. Note similar issues in comment above!

**Comment by JmillsExpensify** [comment]
Sorry ya'll. Missed this in the shuffle. Upwork job is here: https://www.upwork.com/jobs/~01b51a6599d08a678a

**Comment by JmillsExpensify** [comment]
Actually, I missed a comment from @parasharrajat in a related issue https://github.com/Expensify/App/issues/10939. I'm going to go ahead and close out the related Upwork job as well as this issue in favor of the existing, older issue linked.

**Comment by JmillsExpensify** [comment]
Opening this back up now that the navigation refactor is done.

**Comment by JmillsExpensify** [comment]
Still working on testing the remaining issues.

**Comment by mountiny** [comment]
I agree with your findings, seems to be fixed now.

**Comment by JmillsExpensify** [comment]
Ok cool! I'll add this to fixed and we can issue payment once all issues are tested.

**Comment by JmillsExpensify** [comment]
Still working through all the testing issues.

**Comment by JmillsExpensify** [comment]
Putting this back on hold for internal tracking

**Comment by JmillsExpensify** [comment]
Competing priorities.

**Comment by JmillsExpensify** [comment]
Same same. I'm going to look for some help on this one!

**Comment by JmillsExpensify** [comment]
Still trying to make time. Low priority.

**Comment by luacmartins** [comment]
on hold, not overdue

**Comment by JmillsExpensify** [comment]
What he said.

**Comment by JmillsExpensify** [comment]
Same

**Comment by JmillsExpensify** [comment]
Same

**Comment by luacmartins** [comment]
I agree. Closing.

**Comment by aneequeahmad** [comment]
@luacmartins i think reporting bonus payment is pending for this task

**Comment by luacmartins** [comment]
Oh sorry I missed this. Reopening issue. @JmillsExpensify can you help with report bonus payment?

**Comment by aneequeahmad** [comment]
@JmillsExpensify any update on payment ?

**Comment by JmillsExpensify** [comment]
@aneequeahmad Would you mind re-testing this issue, including video, and comment confirming whether the issue still persists?

**Comment by aneequeahmad** [comment]
@JmillsExpensify This issue isn’t reproducible. Fixed in production. I think time to close this issue.

**Comment by JmillsExpensify** [comment]
@aneequeahmad similar to your other reported issues, can you please retest this issue with the exception of the last step and confirm that all is good? At that point we can get you paid for reporting what we've since fixed.

**Comment by aneequeahmad** [comment]
@JmillsExpensify this issue is fixed and doesn't exist in production. We are good to close this and issue payment. Thanks

**Comment by JmillsExpensify** [comment]
Can you please leave a video confirming? Thanks

**Comment by JmillsExpensify** [comment]
Once we get a final test confirmation, we can close this one out.

**Comment by JmillsExpensify** [comment]
Still waiting on final test confirmation

**Comment by aneequeahmad** [comment]
@JmillsExpensify Sorry for the delay. Here is the test confirmation video. All the `Actions are performed` as stated in repro steps [here](https://github.com/Expensify/App/issues/10989#:~:text=Action%20Performed%3A,on%20close%20button)

https://github.com/Expensify/App/assets/36359331/3266ee0d-c088-4c87-aba3-5177047e40c4

**Comment by aneequeahmad** [comment]
@JmillsExpensify lets close this issue as well confirmation video is attached. Thanks

**Comment by JmillsExpensify** [comment]
Nice, thank a bunch! That looks great. Closing this out on the Upwork side via payment.

**Comment by JmillsExpensify** [comment]
Offer sent. Please accept so I can issue payment.

**Comment by aneequeahmad** [comment]
@JmillsExpensify accepted the offer. Thanks for mentioning

**Comment by JmillsExpensify** [comment]
All paid out. Thanks!
